### DISCUSSION TITLE
How to integrate distrobox with nix home-manager?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [89luca89/distrobox#1368] [Q&A]
I'm trying to create an arch linux image, but it's not being able to pull the dotfiles because they are being managed by nix home-manager

![image](https://github.com/89luca89/distrobox/assets/98482469/7f9fb58f-73cb-42d3-8d21-9d92d72f2347)

**Comment by RashiqAzhan** [comment]
Would also like to know.
